src/routes/event-match.tsx

```diff
@@ -91,28 +94,58 @@ const EventMatch = ({ eventKey, matchKey }: Props) => {
     [match],
   )
 
+  if (!isData(match)) {
+    return (
+      <Page
+        back={`/events/${eventKey}`}
+        name={
+          (m === null
+            ? matchKey + ' - '
+            : m.group + (m.num ? ' Match ' + m.num : '') + ' - ') +
+          (event ? event.name : eventKey)
```

Comment:
Can we make this a variable so we don't have to duplicate it between this section and line 131?